x/ibc/light-clients/06-solomachine/client/cli/tx.go

```diff
@@ -25,11 +25,11 @@ const (
 // NewCreateClientCmd defines the command to create a new solo machine client.
 func NewCreateClientCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "create [client-id] [sequence] [path/to/consensus_state.json]",
+		Use:     "create [client-id] [sequence] [path/to/public-key.json] [diversifier] [timestamp]",
```

Comment:
ah my bad I forgot to add it to the command

Reply:
Technically diversifier is in the consensus state. I think this format is probably just easier to interact with